draft-ietf-scitt-architecture.md

```diff
@@ -274,7 +260,7 @@ Transparency Services use `sub` to identify the entity about which they are issu
 Transparency Service:
 
 : an entity that maintains and extends the Append-only Log, and endorses its state.
-A Transparency Service MAY implement a Registration Policy, often referred to by its synonym Notary.
+A Transparency Service MAY alter the contents of the unprotected header of Signed Statements, this will alter the binary content stored on the log, from which receipts are produced.
```

Comment:
I am not clear about this statement! Yes, Transparency Service alters the Unprotected Header of Signed Statements by inserting Receipts, however Receipts are not stored in the log?

Reply:
as a general rule, you never know if anyone has altered an unprotected header... thats what unprotected means.

we do know that a "receipt" will be for whatever "bytes" "go into the log"... in many cases, that means just hashing, and then constructing a list of hashes (for RFC9162)... the details for this are described in the choice of verifiable data structure.

Reply:
No, that is not correct. TS produce receipts and not Transparent Statements